exchange/index_test.go

```diff
@@ -257,6 +257,46 @@ func TestIndexDropRef(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestIndexUpdateRef(t *testing.T) {
+	ds := dss.MutexWrap(datastore.NewMapDatastore())
+
+	idx, err := NewIndex(ds)
+	require.NoError(t, err)
+
+	// create a ref with 1 Key (out of 2) & add it to the index
+	c := blockGen.Next().Cid()
+	ref := &DataRef{
+		PayloadCID:  c,
+		PayloadSize: 256000,
+		Keys:        [][]byte{bytes.NewBufferString("data1").Bytes()},
```

Comment:
```suggestion
		Keys:        [][]byte{[]byte("data1")},
```
That works too no?

Reply:
Right I forgot there was this better way